packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -7,6 +7,19 @@ const ImPropTypes = require("react-immutable-proptypes");
 
 const githubUrl = "https://github.com/devtools-html/debugger.html/blob/master";
 
+const featureLabels = {
+  watchExpressions: "Watch Expressions",
+  chromeScopes: "Chrome Scopes",
+  eventListeners: "Event Listeners",
+  codeCoverage: "Code Coverage",
+  verticalLayout: "Vertical Layout"
+};
```

Comment:
hmm, I think we should probably move this to the the config... that way it's all in one place:

```json
{

  "features": {
     "watchExpressions": {
       "label": "Watch Expressions",
       "enabled": true
     }
}
```

this will likely mean updating devtools-config, feature